code/game/objects/items/storage/belt.dm

```diff
@@ -75,18 +75,17 @@
 	new /obj/item/stack/cable_coil(src,30,pick("red","yellow","orange"))
 
 /obj/item/storage/belt/utility/full/engi/PopulateContents()
-	new /obj/item/screwdriver(src)
-	new /obj/item/wrench(src)
+	new /obj/item/screwdriver/power/cheap(src)

```

Comment:
modularize if possible, make comments if not.

Reply:
. = ..()
new /obj/item/screwdriver/power/cheap(src)
qdel(locate(/obj/item/wrench) in src)

Reply:
messy but probably works